ui/apps/platform/src/Components/PatternFly/TableErrorComponent.tsx

```diff
@@ -17,7 +17,7 @@ function TableErrorComponent({ error, message }: TableErrorComponentProps) {
                 headingLevel="h2"
                 title={getAxiosErrorMessage(error)}
                 icon={ExclamationCircleIcon}
-                iconClassName="pf-v6-u-danger-color-100"
+                iconClassName="pf-v6-u-text-color-status-danger"
```

Comment:
Observation, not change request from difference between this and next change:

It looks like `icon` prop of `EmptyStateIcon` prevented icon element with color, but I do not even see it in current docs page. As follow up, we might be able to rewrite more similar to elsewhere.
https://www.patternfly.org/components/empty-state